app/src/main/java/com/dscout/membranevideoroomdemo/MainActivity.kt

```diff
@@ -66,11 +67,12 @@ class MainActivity : AppCompatActivity() {
         showSystemUi = true,
     )
     @Composable
-    fun MainContent(onConnect: (String, String) -> Unit = { _, _ -> }) {
+    fun MainContent(onConnect: (String, String, Params?) -> Unit = { _, _, _ -> }) {
 
         val scrollableState = rememberScrollState()
         val roomName = remember { mutableStateOf(TextFieldValue("room")) }
         val displayName = remember { mutableStateOf(TextFieldValue("Android User")) }
+        val params = mapOf<String, Any>("token" to "mocktoken")
```

Comment:
put the mocked params in RoomViewModel, that way you don't have to modify MainActivity and RoomActivity
concerns like auth tokens should be in ViewModels

Reply:
I made a change, can you check and let me know if this is what you meant